src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace CommunityToolkit.Maui.MediaElement;
+
+/// <summary>
+/// With MediaElement you can play multimedia inside of your app.
+/// </summary>
+public interface IMediaElement
+{
+	/// <summary>
+	/// The current position of the playing media.
+	/// </summary>
+	TimeSpan Position { get; set; }
+
+	/// <summary>
+	/// The total duration of the loaded media.
+	/// </summary>
+	/// <remarks>Might not be available for some types, like live streams.</remarks>
+	TimeSpan Duration { get; }
```

Comment:
What do you think if we remove it from this interface?
To follow interface segregation principle. 
We may have ILiveStreamMediaElement and IFileSourceMediaElement.

Reply:
@VladislavAntonyuk how would you like to see this implemented?